packages/shared/src/components/modals/NewCommentModal.tsx

```diff
@@ -144,114 +149,79 @@ export default function NewCommentModal({
       setSendingComment(false);
     }
   };
+  const { onInitializeMention } = useUserMention({
```

Comment:
@rebelchris @sshanzel This does not seem to work and I had a brain freeze 🥶  trying to make it work. Can you help me fix it 🙏

Reply:
Thanks for the suggestion @rebelchris